frontend/src/components/single-article/style.css

```diff
@@ -813,7 +810,7 @@ with similar viewport measurments ***************************
     height: 16px;
   }
 
-  .article-content .date-and-author {
+  .article-content {
```

Comment:
Same thing goes for this but this is for the TABLET version: This may affect the whole article-content styling (already defined earlier on Line 761). So may good idea to delete this altogether since this was originally supposed to format specifically for the date and author section which we have since removed.